### DISCUSSION TITLE
Placeholder indicator won't highlight in an empty container.

### DISCUSSION THREAD

**Discussion by uwieske** [GrapesJS/grapesjs#3864] [Q&A]
Hi (artf),

I have been searching for similar issue in the issue database and was hoping to get more practical/concrete or helpful information so that I could resolve my placeholder issue. But, still not successful. 

I have defined my components (with corresponding blocks). The content of my containers (DIV tags) are all empty.
But with an empty setup (with no styling/class annotations on the DIV element), I am not able to insert blocks (components) in the DIV container. The green marker/placeholder won't snap to the container, unless I define a padding style with a height/min-height of 20px. I read the latter in another posted issue. But, if I defined these padding just to resolve placement of block/components, this style properties will also become part of my actual page layout. I don;t desire this behaviour. Is there another solution? 
I noticed that a container must have a minimum height in order the placeholder to work properly.
Any suggestions?

**Comment by artf** [comment]
You can create a custom empty state for your DOM elements in the canvas (this will not affect the final code)
https://github.com/artf/grapesjs/issues/3076#issuecomment-714758094
